hyo2/soundspeed/formats/readers/aml.py

```diff
@@ -153,6 +154,11 @@ def _csv_header(self):
                     except ValueError:  # it may be no-lock
                         lon = None
                     continue
+                
+                if tokens[0].lower() == self._aml.serial2:
+                    if tokens[1]:
+                        self.ssp.cur.meta.sn = tokens[1]
```

Comment:
Here a [strip](https://docs.python.org/3/library/stdtypes.html?highlight=strip#str.strip) call is missing.